components/elections/constituency-lookup/index.js

```diff
@@ -32,17 +35,29 @@ const ConstituencyLookup = ({
       value: id,
       display: candidateName,
       displayConstituency: constituencyName,
+      partyName,
       type: 'candidate',
     })),
   ];
 
-  const RenderSuggestion = ({ display, type, displayConstituency }) => (
-    <div className={`suggestion-entry suggestion-entry--${type}`}>
-      {type === 'candidate' && <i className="candidate-icon" />}
-      {display}
-      {type === 'candidate' && <div className="candidate-constituency">{displayConstituency}</div>}
-    </div>
-  );
+  const RenderSuggestion = ({ display, type, displayConstituency, partyName }) => {
+    const { color } = type === 'candidate' ? getPartyInfo(partyName) : { color: 'black' };
```

Comment:
I think this is clearer:

```suggestion
    const color = type === 'candidate' ? getPartyInfo(partyName).color : 'black';
```